framework/include/materials/MaterialPropertyInterface.h

```diff
@@ -164,6 +165,8 @@ class MaterialPropertyInterface
   bool getMaterialPropertyCalled() const { return _get_material_property_called; }
 
 protected:
+  /// Parameters of the object with this interface
+  const InputParameters & _mi_params;
```

Comment:
Do we need to do this stuff anymore, should MooseInterface do this in the constructor:  `_interface_params(this_object->parameters())`, so interface_params would just be available for all child objects.